payroll/models/res_config_settings.py

```diff
@@ -35,3 +35,9 @@ class ResConfigSettings(models.TransientModel):
         help="Require rule.code, rule.category, category.code, structure.code",
         default=False,
     )
+    worked_days_from_prev_month = fields.Boolean(
+        config_parameter="payroll.worked_days_from_prev_month",
+        string="Fetch worked days and leaves from the previous month",
+        help="Pay fixed items for this month, with adjustments for previous month",
+        default=False,
```

Comment:
I don't understand what this means. For example, if WORK100 for this month is 20 days and also 20 days for the previous month will the WORK100 payslip input line have 40 days? Also, the help message doesn't make this any clearer. Please consider re-wording this to make it clearer what the purpose of this setting is.

Reply:
So if the option is active, WORK100 will show days worked in the previous month only.